exec_func.go

```diff
@@ -47,10 +47,12 @@ func InsertOnDuplicateKeyUpdateFunc(scope *gorm.Scope, columnNames, groups []str
 	var duplicateUpdates []string
 
 	for i := range columnNames {
-		duplicateUpdates = append(
-			duplicateUpdates,
-			fmt.Sprintf("%s = VALUES(%s)", columnNames[i], columnNames[i]),
-		)
+		if columnNames[i] != "`created_at`" {
```

Comment:
I would prefer if the [happy path was kept to the left](https://medium.com/@matryer/line-of-sight-in-code-186dd7cdea88) both since that's how the rest of the code is written but also because I think it's more readable. It would also be a good idea if there were more special cases in the future.

```go
if columnNames[i] != "`created_at`" {
    continue
}

// Move on and add columnn...
```

Reply:
Done